### ISSUE TITLE
[BUG]: need more user visibility into update progress/ button to trigger update

### ISSUE THREAD

**Issue by r-barton** [KittyCAD/modeling-app#3953]
### Describe the bug

When using the windows desktop app, updates only happen once the app is closed. On app close windows UAC prompts if Zoo Modeling app should make changes to the computer. There is no other indication of updates available or why the UAC window is appearing.

### Steps to Reproduce

1. Start windows modeling app from an older release (e.g. v.0.25.2)
2. Close the app

### Expected Behavior

I expect that there is a button or other indication that a new version of the modeling app is available. I believe this has only been happening since the switch to electron

### Desktop OS

Windows 10

### Version

v0.25.2 -> v0.25.3

**Comment by jessfraz** [comment]
yeah this is bad across the board i know @franknoirot and @pierremtb have ideas

**Comment by jessfraz** [comment]
slack thread here https://kittycadworkspace.slack.com/archives/C04KFV6NKL0/p1727116938603349

**Comment by pierremtb** [comment]
First step toward making this better was #3902, should be in the next release (so users will see it once the next next release is out)

**Comment by franknoirot** [comment]
A next step is #3994, which will show the download progress so people know the download is progressing even if it takes a while.

**Comment by franknoirot** [comment]
Actually a first step, I think, that we can do is add a native app menu item that allows users to "Check for updates" whenever. That's more important than the downloader progress.

**Comment by pierremtb** [comment]
@franknoirot  Agreed! I mentioned something on Slack the other day about adding something in the LowerRightControls. Could be both a button and a progress indicator potetially?

**Comment by franknoirot** [comment]
> [@franknoirot](https://github.com/franknoirot) Agreed! I mentioned something on Slack the other day about adding something in the LowerRightControls. Could be both a button and a progress indicator potetially?

Yes I recall that. I like that idea, since we already have existing patterns for the LowerRightControls with toggle-indicators.

I think the crux of this bug is fixed as of #3902, since Windows (and all platforms) users now never see a permissions request before seeing an in-app popup. I have [a PR up](https://github.com/KittyCAD/modeling-app/pull/5607) for [this comment](https://github.com/KittyCAD/modeling-app/issues/3953#issuecomment-2672441192), and #3994 and your idea are really nice-to-haves beyond those behaviors. So I'm going to close this bug as complete.
